views/attendance/attendanceREADME.md

```diff
@@ -0,0 +1,9 @@
+#Attendance Documentation
```

Comment:
Need to throw a space in front of `#` for markdown to render those headers (at least on GitHub specifically)